lib/src/main/java/com/auth0/jwt/algorithms/NoneAlgorithm.java

```diff
@@ -24,4 +24,8 @@ public void verify(DecodedJWT jwt) throws SignatureVerificationException {
         return new byte[0];
     }
 
+    @Override
+    public byte[] sign(byte[] contentBytes) throws SignatureGenerationException {
```

Comment:
shouldn't this method also be annotated as `@deprecated` ?

Reply:
@Deprecated added